docs/extend/frontend.md

```diff
@@ -105,13 +195,13 @@ You should develop extensions with debug mode turned **on** in `config.php`. Thi
 
 ## Changing the UI
 
-Flarum's interface is constructed using a JavaScript framework called [Mithril.js](https://mithril.js.org/archive/v0.2.5). If you are familiar with [React](https://reactjs.org), then you'll catch on in no time. But if you are not familiar with any JavaScript frameworks, we suggest you go through a [tutorial](http://ratfactor.com/mithril1/?/shire) to understand the fundamentals before proceeding.
+Flarum's interface is constructed using a JavaScript framework called [Mithril.js](https://mithril.js.org/). If you are familiar with [React](https://reactjs.org), then you'll catch on in no time. But if you are not familiar with any JavaScript frameworks, we suggest you go through a [tutorial](https://mithril.js.org/simple-application.html) to understand the fundamentals before proceeding.
 
 The crux of it is that Flarum generates virtual DOM elements which are a JavaScript representation of HTML. Mithril takes these virtual DOM elements and turns them into real HTML in the most efficient way possible. (That's why Flarum is so speedy!)
 
 Because the interface is built with JavaScript, it's really easy to hook in and make changes. All you need to do is find the right extender for the part of the interface you want to change, and then add your own virtual DOM into the mix.
 
-Every mutable part of the interface is really just a *list of items*. For example:
+Most mutable parts of the interface are really just *list of items*. For example:
```

Comment:
Removing the "a" before "list of items" leaves this phrase in a half-English state. Did you mean "are really just list**s** of items"?